src/FunctionApps/python/GenerateCSVs/__init__.py

```diff
@@ -1,47 +1,54 @@
-import csv
-import io
-
-import azure.functions as func

```

Comment:
Why is the git diff having a fit over this 🤷

Reply:
`dos2unix` - azure's templates all (surprise!) have windows line endings